test/test_helper.exs

```diff
@@ -3,3 +3,6 @@
 ExUnit.start
 
 Ecto.Adapters.SQL.Sandbox.mode(CoursePlanner.Repo, :manual)
+
+Code.compiler_options(ignore_module_conflict: true)
```

Comment:
To supress a warning during test that the module was being loaded twice.
But now that I'm testing again, I don't have to load it here in the next line, so I'm removing it.